starter/script.js

```diff
@@ -1 +1,34 @@
-// put code here!
+confirm("This game contains strong language and some adult situations. Player discretion is advised...")
+
+confirm("Welcome to West Baltimore");
+
+var response = prompt("Wanna be in the game shorty? Yes or No?");
+
+if (response === "No") {
+  alert("You can't lose if you don't play. [Technical, but less fun, victory]");
+  //how do you fully stop code from running at this point?
+} else {
+  alert("Ight young hoppa, I'll put you in charge of the count.");
+};
+
+var count = prompt("It's the end of the day shorty. Is the COUNT RIGHT or did you FUCK UP THE COUNT?");
```

Comment:
you could move line 14 - 29 inside of the else statement of the condition above. why would you want to to do that? one reason is that if you answered no to the first question, the program would no longer ask you any questions. that's how nested conditionals can be used. what are nested conditionals? if/else/else if statements inside of other if/else/else if statements